CMakeLists.txt

```diff
@@ -160,7 +160,8 @@ ExternalProject_Add(protobuf
     -DBUILD_SHARED_LIBS:STRING=no
     -DCMAKE_INSTALL_LIBDIR:STRING=lib
     -DCMAKE_INSTALL_PREFIX:PATH=${TRITON_THIRD_PARTY_INSTALL_PREFIX}/protobuf
-  PATCH_COMMAND python3 ${CMAKE_CURRENT_SOURCE_DIR}/tools/install_src.py --src <SOURCE_DIR> ${INSTALL_SRC_DEST_ARG}
+  PATCH_COMMAND git cherry-pick -n f180289c4670ca1afde5865bb8a7f2b61a3efcc5
```

Comment:
what's the likelihood of this generating a conflict in the future if we upgrade our version of grpc again? I think low to none since the we won't need to cherry pick a change across a major version?

Reply:
We don't need this patch if protobuf version is advanced, so we just drop this once we upgrade GRPC again (which should pick up newer version of protobuf)